RestImageResize/OpenWaveRestApiEncoder.cs

```diff
@@ -110,5 +116,17 @@ public Uri EncodeUri(Uri uri)
             Url url = ImageTransformationService.GetTransformedImageUrl(Url.Parse(uri.ToString()), transformBuilder);
             return new Uri(url.ToString(), UriKind.RelativeOrAbsolute);
         }
+
+        private void EnsureAuthorizedQuery(ImageTransformQuery query)
+        {
+            var authorized = QueryAuthorizer.IsAuthorized(query);
+
+            if (!authorized)
+            {
+                throw new HttpException((int) HttpStatusCode.Forbidden,
+                    string.Format("Forbidden query 'w={0}&h={1}&t={2}&h={3}'",
```

Comment:
They help to debug. And we're not exposing anything internal: these are parameters from request.